app/src/main/java/net/gsantner/markor/activity/DocumentEditAndViewFragment.java

```diff
@@ -159,34 +162,6 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         _lineNumbersView.setup(_hlEditor);
         _lineNumbersView.setLineNumbersEnabled(_appSettings.getDocumentLineNumbersEnabled(_document.path));
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP && _appSettings.getSetWebViewFulldrawing()) {
```

Comment:
same code moved into block which is called when webview is actually required and loaded